package-res/resources/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -213,6 +213,10 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
             return true;
           }
 
+          if(dateFormat.match(/mm/)) {
+            return true;
+          }
+
```

Comment:
dont think fixing it to yy-mm-dd is the right solution. Things like y-mm-dd, yy-m-d are also formats that in dojo date picker dont make sense because they will be picking minutes. I think if we find the existence of the m string on our use case is probably the best solution to cover all the cases. If we mark this as legacy, then the current conversion method will deal with it